app/Http/Controllers/Api/ChantierEquipmentSyncController.php

```
@@ -0,0 +1,182 @@
+                // Check for existing open tracking
+                $existing = ChantierEquipmentTracking::where('trackable_type', $trackableType)
+                    ->where('trackable_id', $trackable->id)
+                    ->whereNull('check_out_at')
+                    ->first();
+
+                if ($existing) {
+                    // Auto check-out from previous chantier
+                    $existing->update(['check_out_at' => now()]);
+                }
+
+                $tracking = ChantierEquipmentTracking::create([
```

PHPStan (2.2.8)

[error] Call to an undefined static method App\Models\Chantiers\ChantierEquipmentTracking::create().


(staticMethod.notFound)